kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -129,11 +131,21 @@
           facility: this.facility,
         };
         this.updateUser(this.userid, payload, this.role_new);
+        // if logged in admin updates role to learner, redirect to learn page
+        if (Number(this.userid) === this.session_user_id) {
```

Comment:
We should probably also add a warning message that an Admin will no longer be able to access the management page to turn themselves back into an Admin once they become a Learner (post-MVP)